htdocs/product/stock/replenish.php

```diff
@@ -194,15 +194,22 @@
 							// TODO Get desc in language of thirdparty
 						}
 
+						// If we use multicurrency
+						if (isModEnabled('multicurrency') && !empty($productsupplier->fourn_multicurrency_code) && $productsupplier->fourn_multicurrency_code != $conf->currency) {
+							$line->multicurrency_code = $productsupplier->fourn_multicurrency_code;
+							$line->fk_multicurrency = $productsupplier->fourn_multicurrency_id;
+							$line->multicurrency_subprice = $productsupplier->fourn_multicurrency_unitprice;
+						}
+
 						$line->tva_tx = $productsupplier->vatrate_supplier;
 						$line->subprice = $productsupplier->fourn_pu;
 						$line->total_ht = $productsupplier->fourn_pu * $qty;
 						$tva = $line->tva_tx / 100;
 						$line->total_tva = $line->total_ht * $tva;
 						$line->total_ttc = $line->total_ht + $line->total_tva;
 						$line->remise_percent = $productsupplier->remise_percent;
-						$line->ref_fourn = $productsupplier->ref_supplier;
-						$line->type = $productsupplier->type;
```

Comment:
This is just to fix deprecated properties, not related with the core of the PR.

Reply:
Fixing of deprecated properties must be done into develop only. Can you submit fix with just fix of visible bug ?

Reply:
@eldy Sure, I just did.